kazoo/tests/test_utils.py

```diff
@@ -30,6 +30,24 @@ def test_timeout_arg(self):
                     timeout = call_args[0][1]
                     assert timeout >= 0, "socket timeout must be nonnegative"
 
+    def test_ssl_server_hostname(self):
```

Comment:
If possible, I probably would have liked a test around the new `check_hostname` param. What do you think?

Reply:
Good idea, test added